config/templating/templating.go

```diff
@@ -76,6 +76,13 @@ var platformTemplatingMap = map[string]map[string]string{
 	platform.CloudStackConfigDrive: {
 		fieldHostname: "CLOUDSTACK_LOCAL_HOSTNAME",
 	},
+	platform.Custom: {
+		fieldHostname:  "HOSTNAME",
```

Comment:
Should the variable names be namespaced like the others?

Reply:
I was unsure. From an implementation standpoint, it makes a lot of sense. From a user I think it makes more sense to use what they are using in their templated config so they don't have to mental map them. It's just a little bit more they need to understand (although it is documented below). Either way I don't feel strongly.